shuffle/cli/src/shared.rs

```diff
@@ -428,9 +426,12 @@ impl Default for Network {
 /// Generates the typescript bindings for the main Move package based on the embedded
 /// diem types and Move stdlib. Mimics much of the transaction_builder_generator's CLI
 /// except with typescript defaults and embedded content, as opposed to repo directory paths.
-pub fn generate_typescript_libraries(project_path: &Path) -> Result<()> {
+pub fn codegen_typescript_libraries(
+    project_path: &Path,
+    publishing_address: &AccountAddress,
+) -> Result<()> {
     let pkg_path = project_path.join(MAIN_PKG_PATH);
-    let _compiled_package = build_move_package(&pkg_path)?;
+    let _compiled_package = build_move_package(&pkg_path, publishing_address)?;
```

Comment:
nit: can we just make this line `build_move_package(&pkg_path, publishing_address)?;` instead of assigning it to a variable we don't use